drivers/SmartThings/zwave-switch/profiles/zooz-zen-30-dimmer-relay.yml

```diff
@@ -0,0 +1,285 @@
+name: zooz-zen-30-dimmer-relay
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 2
+  - id: switchLevel
+    version: 1
+  - id: button
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: Switch
+- id: switch1
+  capabilities:
+  - id: switch
+    version: 2
```

Comment:
why did you use version 2?

Reply:
it was just experimenting out of curiosity (forgot to change to 1 version back)